addons/website_sale/static/src/js/checkout.js

```diff
@@ -188,33 +200,21 @@ publicWidget.registry.websiteSaleCheckout = publicWidget.Widget.extend({
     },
 
     /**
-     * Hide the selected pickup location.
+     * Hide the pickup location and reset the order location name and address if defined.
```

Comment:
where do you reset the name in this function?

Reply:
Good catch! I don't reset anymore since we decided to keep the pickup location when you switch delivery methods.